lrs/models.py

```diff
@@ -312,6 +312,15 @@ def get_queryset(self):
 
 
 class Activity(models.Model):
+
+    class Meta:
+        indexes = [
+            models.Index(
+                fields=["activity_id", "canonical_version"],
+                condition=models.Q(canonical_version__isnull=False),
+                name="ix_activity_canonical_version"
```

Comment:
maybe here should be: **is**_activity_canonical_version  instead of `ix_` ?

Reply:
It was deliberate, based on some stuff I googled about index naming conventions - `ix` for **i**nde**x**.

Thinking about it more, I should probably keep the name consistent with the manually applied index: `idx_canonical_activity` so I'll change it to that

Reply:
done